launch/client.py

```diff
@@ -813,16 +813,16 @@ def create_model_bundle_from_triton_enhanced_runnable_image_v2(
         repository: str,
         tag: str,
         command: List[str],
-        healthcheck_route: Optional[str],
-        predict_route: Optional[str],
+        healthcheck_route: Optional[str] = None,
+        predict_route: Optional[str] = None,
         env: Dict[str, str],
         readiness_initial_delay_seconds: int,
         triton_model_repository: str,
-        triton_model_replicas: Optional[Dict[str, str]],
+        triton_model_replicas: Optional[Dict[str, str]] = None,
```

Comment:
Huh I take it we don't have any prior such bundles then? Otherwise I'm guessing people would have run into this.